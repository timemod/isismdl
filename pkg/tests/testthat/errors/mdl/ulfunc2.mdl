ul_function eta(x, y);
ident z = eta(x > 2, y);
