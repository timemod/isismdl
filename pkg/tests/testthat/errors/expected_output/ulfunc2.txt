Isis Model Compiler 3.00
Compiling model ...
       1 equations processed
Ordering equations ...
Writing MIF file ...
Writing cross-reference file ...
End compilation
Reading mif file ...
Model with     1 equations read
Checking Model-code ...
Argument of user language function is a logical in equation z
Errors encountered in model ...
